src/Resources/contao/classes/DataContainer.php

```diff
@@ -902,8 +902,9 @@ protected function initPicker()
 
 		$this->setPickerValue();
 
-		$strDriver = 'DC_' . $GLOBALS['TL_DCA'][$this->strPickerTable]['config']['dataContainer'];
-		$objDca = new $strDriver($this->strPickerTable);
+		// Create a DataContainer object (see #886)
+		$objDca = new \DataContainer();
```

Comment:
This will be an issue because the `onload_callback` is no longer executed… And no, it's not executed in the real table, because most of the time the `strPickerTable` will be different for the popup.